README.md

```diff
@@ -19,35 +19,51 @@ Medplum is a developer platform that enables flexible and rapid development of h
-  - [First time setup](#first-time-setup)
 
 ## Contributing
 
 **We heartily welcome any and all contributions that match our engineering standards!**
 
-That being said, this codebase isn't your typical open source project because it's not a library or package with a limited scope -- it's our entire product.
+That being said, this codebase isn't your typical open source project because it's not a library or package with a
+limited scope -- it's our entire product.
 
 ### Ground Rules
 
 #### Contributions and discussion guidelines
 
 By making a contribution to this project, you are deemed to have accepted the [Developer Certificate of Origin](https://developercertificate.org/) (DCO).
 
-All conversations and communities on Medplum agree to GitHub's [Community Guidelines](https://help.github.com/en/github/site-policy/github-community-guidelines) and [Acceptable Use Policies](https://help.github.com/en/github/site-policy/github-acceptable-use-policies). This code of conduct also applies to all conversations that happen within our contributor community here on GitHub. We expect discussions in issues and pull requests to stay positive, productive, and respectful. Remember: there are real people on the other side of that screen!
+All conversations and communities on Medplum are expected to follow GitHub's [Community Guidelines](https://help.github.com/en/github/site-policy/github-community-guidelines)
+and [Acceptable Use Policies](https://help.github.com/en/github/site-policy/github-acceptable-use-policies). We expect
+discussions in issues and pull requests to stay positive, productive, and respectful. Remember: there are real people on
+the other side of the screen!
 
-#### Reporting a bug or discussing a feature idea
+#### Reporting a bug or proposing a new feature
 
-If you found a technical bug on Medplum or have ideas for features we should implement, the issue tracker is the best place to share your ideas. Make sure to follow the issue template and you should be golden! ([click here to open a new issue](https://github.com/medplum/medplum/issues/new))
+If you found a technical bug on Medplum or have ideas for features we should implement, the issue tracker is the best
+place to . ([click here to open a new issue](https://github.com/medplum/medplum/issues/new))
+
+### Writing documentation or blog content
+
+Did you learn how to do something using Medplum that wasn't obvious on your first try? By contributing your new knowledge
+to our documentation, you cna help others who might have a similar use case!
+
+Our documentation is hosted on [medplum.com/docs](/docs), but it is built from [Markdown](https://www.markdownguide.org/)
+files in our [`docs` package](https://github.com/medplum/medplum/tree/main/packages/docs/docs).
+
+For relatively small changes, you can edit files directly from your web browser on [Github.dev](https://github.dev/medplum/medplum/blob/main/packages/docs/docs/home.md)
+without needing to clone the repository.
```

Comment:
Consider referencing our "Contribute without coding" issue?  https://github.com/medplum/medplum/issues/2249

Reply:
Most of the copy in that issue is reproduced in the Contributing docs page, but I'll link to that!